### ISSUE TITLE
TZP: "script defaults" metric catches a "fontList.match(...) is null" error

### ISSUE THREAD

**Issue by Thorin-Oakenpants** [sereneblue/chameleon#582]
# Prerequisites

Please use issues for bugs only! Answer the following questions for yourself before submitting an issue: **YOU MAY DELETE THE PREREQUISITES SECTION.**

- [x] I am running the latest version
- [x] I checked the documentation and found no answer
- [x] I checked to make sure that this issue has not already been filed

## Expected Behavior

no errors: values are defaults for: proportional-font + monospace size + sans-serif size + serif size

![expected](https://github.com/sereneblue/chameleon/assets/16656956/de8e329b-706a-47de-a50f-3c8bdf10f9b7)

as seen in Firefox> Settings > Language and Settings > Fonts > Advanced > per script in the dropdown
- for example here is `devanagari` which is `sans-serif-13-16-16` as per the expected example above

![settings](https://github.com/sereneblue/chameleon/assets/16656956/0f261a24-5102-4521-bb4b-d18dcc4b70e5)

## Current Behavior

https://arkenfox.github.io/TZP/tzp.html#fonts > `script defaults` metric
- "TypeError: can't access property \"map\", fontList.match(...) is null"
- https://github.com/sereneblue/chameleon/blob/03e3a91c04fb10b541954ee49a1c88bf94fd0e47/src/lib/spoof/font.ts#L73
- [code used](https://github.com/arkenfox/TZP/blob/ea4631084254ed1e5b7342858ad65151a1108ab4/js/fonts.js#L1402-L1434)
   - basically it loops a bunch of writing scripts by setting a language tag on an element, and for each of those loops three generic font-families (monospace, serif, sans-serif). I actually use two elements for technical reasons, but that's not important

## Relevant settings

enable `Spoof font fingerprint`

## Context (Environment)

all of them

**Comment by sereneblue** [comment]
Thanks, I'll review.

**Comment by Thorin-Oakenpants** [comment]
> Thanks, I'll review.

🍕

**Comment by Thorin-Oakenpants** [comment]
verified fixed ... 🧀
